src/reducers/index.js

```diff
@@ -13,8 +13,10 @@ export default (state = initialState, action) => {
   switch (action.type) {
     case INCREMENT:
     // Fill in the body of this case
+      return { count: state.count + 1 }
```

Comment:
Another way of updating your state with the `prefix` operator
```suggestion
      return { count: ++state.count  }
```